static/usage/v8/action-sheet/role-info-on-dismiss/angular/example_component_ts.md

```diff
@@ -32,7 +35,8 @@ export class ExampleComponent {
 
   constructor() {}
 
-  logResult(ev) {
+  logResult(event: Event) {
+    const ev = event as CustomEvent<OverlayEventDetail<string>>;
```

Comment:
Couldn't it be like this?
`logResult(event: CustomEvent<OverlayEventDetail<string>>) {
    console.log(JSON.stringify(event.detail, null, 2));
}`

Reply:
Fixed: https://github.com/ionic-team/ionic-docs/pull/3909/commits/03c040ad3a379afefff3b261457b72f4da898a1a